dev-vcs/git/git-2.36.1.recipe

```diff
@@ -0,0 +1,337 @@
+It outclasses SCM tools like Subversion, CVS, Perforce, and ClearCase with \
+features like cheap local branching, convenient staging areas, and multiple \
+workflows."
+HOMEPAGE="https://git-scm.com/"
+COPYRIGHT="2005-2020 Git Authors (see git web site for list)"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://www.kernel.org/pub/software/scm/git/git-$portVersion.tar.xz"
+CHECKSUM_SHA256="405d4a0ff6e818d1f12b3e92e1ac060f612adcb454f6299f70583058cb508370"
+SOURCE_URI_2="https://www.kernel.org/pub/software/scm/git/git-manpages-$portVersion.tar.xz"
+CHECKSUM_SHA256_2="b1e6e651333283bfe2abbdf10baa858f61c5ec9d3caacb30f44888d78d964e6f"
+SOURCE_URI_3="https://www.kernel.org/pub/software/scm/git/git-htmldocs-$portVersion.tar.xz"
+CHECKSUM_SHA256_3="f2a44ee1acfc1b8e2ce9a90ae767226c862d5a6596bed0404de70fdef8e67c8d"
+PATCHES="git-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+GLOBAL_WRITABLE_FILES="
+	settings/gitconfig keep-old
+	"
+USER_SETTINGS_FILES="
+	settings/git directory
+	"
+
+PROVIDES="
+	git$secondaryArchSuffix = $portVersion compat >= 2.3
+	cmd:git = $portVersion compat >= 2.3
+	cmd:git_receive_pack = $portVersion compat >= 2.3
+	cmd:git_shell = $portVersion compat >= 2.3
+	cmd:git_upload_archive = $portVersion compat >= 2.3
+	cmd:git_upload_pack = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES="
+		git
+		"
+fi
+PROVIDES_arch="
+	git${secondaryArchSuffix}_arch = $portVersion compat >= 2.3
+	cmd:git_archimport = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_arch="
+		git_arch
+		"
+fi
+PROVIDES_cvs="
+	git${secondaryArchSuffix}_cvs = $portVersion compat >= 2.3
+	cmd:git_cvsimport = $portVersion compat >= 2.3
+	cmd:git_cvsserver = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_cvs="
+		git_cvs
+		"
+fi
+PROVIDES_daemon="
+	git${secondaryArchSuffix}_daemon = $portVersion compat >= 2.3
+	cmd:git_daemon = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_daemon="
+		git_deamon
+		"
+fi
+PROVIDES_email="
+	git${secondaryArchSuffix}_email = $portVersion compat >= 2.3
+	cmd:git_send_email = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_email="
+		git_email
+		"
+fi
+PROVIDES_gui="
+	git${secondaryArchSuffix}_gui = $portVersion compat >= 2.3
+	cmd:git_gui = $portVersion compat >= 2.3
+	cmd:gitk = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_gui="
+		git_gui
+		"
+fi
+PROVIDES_source="
+	git${secondaryArchSuffix}_source = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_source="
+		git_source
+		"
+fi
+PROVIDES_svn="
+	git${secondaryArchSuffix}_svn = $portVersion compat >= 2.3
+	cmd:git_svn = $portVersion compat >= 2.3
+	"
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	REPLACES_svn="
```

Comment:
i think one could use a loop here:
for pkg in arch cvs daemon email gui source svn; do eval "REPLACES_${pkg}=git_${pkg}"; done

Reply:
That worked :)